core/runner/pom.xml

```diff
@@ -69,10 +69,6 @@
       <groupId>org.apache.activemq</groupId>
       <artifactId>activemq-osgi</artifactId>
     </dependency>
-    <dependency>
```

Comment:
Awesome! Thank you @Skejven ! 

Actually it looks like guice is referenced also in *worker* module, but is not used at all there (or am I wrong?).
So we could remove Guice from the worker pom and then also from top-level pom, isn't it?

Then we could remove guice from `feature.xml` file. That would b great, right?

(there will be only some transitive dependencies to guice from our functional tests that use [Bobcat](https://github.com/Cognifide/bobcat)).

Could be done here or in another pull request if you prefer...

Reply:
Thank you @wiiitek for the review.
I will remove Guice dependency in the separate PR.